src/DynamicRupture/Initializer/BaseDRInitializer.cpp

```diff
@@ -1,10 +1,27 @@
 #include "BaseDRInitializer.h"
 
 #include "Initializer/ParameterDB.h"
-#include "Model/common.hpp"
 #include "SeisSol.h"
 #include "generated_code/kernel.h"
+#include <DynamicRupture/Misc.h>
 #include <Eigen/Dense>
+#include <Geometry/MeshDefinition.h>
+#include <Initializer/DynamicRupture.h>
+#include <Initializer/tree/LTSInternalNode.hpp>
+#include <Initializer/tree/LTSTree.hpp>
+#include <Initializer/tree/Layer.hpp>
+#include <Kernels/precision.hpp>
+#include <Numerical_aux/Transformation.h>
+#include <cassert>
+#include <cmath>
+#include <cstddef>
+#include <init.h>
+#include <string>
+#include <tuple>
+#include <unordered_map>
+#include <utility>
+#include <utils/logger.h>
+#include <vector>
```

Comment:
Not a fan of this include list:
* SeisSol Headers should be marked with "", not with <>
* Sorting is quite off. I'd group SeisSol Headers, std lib and external libraries (eigen/util).

Reply:
Same point also holds for several other files.

Reply:
Test-wisely done. Then, also resorted the includes by clang-format, as clang-tidy will complain anyways about that.

Downside: we now modify basically every file in SeisSol by a tiny bit.

Reply:
Consider
```
diff --git a/src/DynamicRupture/Initializer/BaseDRInitializer.cpp b/src/DynamicRupture/Initializer/BaseDRInitializer.cpp
index b0207b60..003ac12d 100644
--- a/src/DynamicRupture/Initializer/BaseDRInitializer.cpp
+++ b/src/DynamicRupture/Initializer/BaseDRInitializer.cpp
@@ -1,27 +1,26 @@
 #include "BaseDRInitializer.h"

-#include "Initializer/ParameterDB.h"
-#include "SeisSol.h"
-#include "generated_code/kernel.h"
-#include <DynamicRupture/Misc.h>
-#include <Eigen/Dense>
-#include <Geometry/MeshDefinition.h>
-#include <Initializer/DynamicRupture.h>
-#include <Initializer/tree/LTSInternalNode.hpp>
-#include <Initializer/tree/LTSTree.hpp>
-#include <Initializer/tree/Layer.hpp>
-#include <Kernels/precision.hpp>
-#include <Numerical_aux/Transformation.h>
 #include <cassert>
 #include <cmath>
 #include <cstddef>
 #include <init.h>
 #include <string>
 #include <tuple>
-#include <unordered_map>
-#include <utility>
+
+#include <Eigen/Dense>
 #include <utils/logger.h>
-#include <vector>
+
+#include "DynamicRupture/Misc.h"
+#include "Geometry/MeshDefinition.h"
+#include "Initializer/DynamicRupture.h"
+#include "Initializer/ParameterDB.h"
+#include "Initializer/tree/LTSInternalNode.hpp"
+#include "Initializer/tree/LTSTree.hpp"
+#include "Initializer/tree/Layer.hpp"
+#include "Kernels/precision.hpp"
+#include "Numerical_aux/Transformation.h"
+#include "SeisSol.h"
+#include "generated_code/kernel.h"

 namespace seissol::dr::initializer {
 void BaseDRInitializer::initializeFault(seissol::initializer::DynamicRupture const* const dynRup,
```

Reply:
It's currently (mostly automatically) done by having the "" includes first, followed by the <> includes.
(I changed the <> to "" manually)

We can switch the order to <> before ""; I have no idea what's more efficient of these orders.